ibm_catalog.json

```diff
@@ -213,6 +214,7 @@
           "name": "static-website-configurator",
           "install_type": "extension",
           "working_directory": "solutions/tf-extension-da",
+          "short_description": "Ideal for hosting static websites on IBM Cloud Object Storage. Configures public access and website settings for an existing bucket as an extension.",
```

Comment:
Please update this -
`as an extension` - not required in description
actual pupose is - Public access for the provided Object Storage bucket